pymc3/distributions/multivariate.py

```diff
@@ -1289,6 +1183,145 @@ def random(self, point=None, size=None):
         return samples
 
 
+def LKJCholeskyCov(
+        name, eta, n, sd_dist, compute_corr=False, name_stds="stds", name_rho="Rho"
+):
+    R"""Wrapper function for covariance matrix with LKJ distributed correlations.
+
+    This defines a distribution over Cholesky decomposed covariance
+    matrices, such that the underlying correlation matrices follow an
+    LKJ distribution [1] and the standard deviations follow an arbitray
+    distribution specified by the user.
+
+    Parameters
+    ----------
+    name: str
+        The name given to the variable in the model.
+    eta: float
+        The shape parameter (eta > 0) of the LKJ distribution. eta = 1
+        implies a uniform distribution of the correlation matrices;
+        larger values put more weight on matrices with few correlations.
+    n: int
+        Dimension of the covariance matrix (n > 1).
+    sd_dist: pm.Distribution
+        A distribution for the standard deviations.
+    compute_corr: bool, default=False
```

Comment:
Possibly better: (?)
```
If `True`, return three values: The cholesky decomposition, the correlation and the standard deviations of the covariance matrix. Otherwise only return the packed cholesky decomposition.
```